code/modules/shuttle/navigation_computer.dm

```diff
@@ -57,6 +60,25 @@
 	else
 		QDEL_NULL(my_port)
 
+/// "Initializes" any default port ids we have, done so add_jumpable_port can be a proper setter
+/obj/machinery/computer/camera_advanced/shuttle_docker/proc/set_init_ports()
+	var/list/init_ports = jump_to_ports.Copy()
+	jump_to_ports = list() //Reset it so we don't get dupes
+	for(var/port_id in init_ports)
+		add_jumpable_port(port_id)
```

Comment:
In the PR description  you said you added 1 vestigial proc, `remove_jumpable_port` which I got

But I don't see this being called anywhere, is it?

Reply:
I uh. I was going to call it in Initialize. but I forgot to use it